test/atomic_enum_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class TestAtomicEnum < MiniTest::Unit::TestCase
```

Comment:
I kinda dislike doing things like that prematurely since I assume they're there for a reason. If I do, someone always dislikes the change (and is usually very vocal about it). But OK, I'll do that after.